.github/workflows/swift-toolchain.yml

```diff
@@ -3676,6 +3680,12 @@ jobs:
       - name: Package SDK
         if: inputs.build_android
         run: |
+          if ("${{ inputs.build_arch }}" -eq "amd64") {
```

Comment:
Is this supposed to be `build_arch` or `host_arch`?

Reply:
To connect the dots ... the reason that we should then use the build architecture is that we are now doing native compilation only after @hjyamauchi's changes?

Reply:
I believe these packages used to be built for the `x86` platform. Let's follow up next week. These changes - along with the `swift-installer-scripts` fix the build for now.